release notes/v0.49.0.md

```diff
@@ -0,0 +1,126 @@
+- [browser#1121](https://github.com/grafana/xk6-browser/pull/1121) fixes `dblClick` so that it works with `onDblClick` and performs two clicks on the specified element.
+- [browser#1152](https://github.com/grafana/xk6-browser/pull/1152) fixes a nil pointer dereference when navigating around on SPA websites.
+
+## Maintenance and internal improvements
+
+- [#3204](https://github.com/grafana/k6/pull/3204) Internal refactor to make future distributed execution work easier. With a small fix to tests in [#3531](https://github.com/grafana/k6/pull/3531). Thanks to @na-- :tada:.
+- A lot of lint fixes through out the k6 code base [#3460](https://github.com/grafana/k6/pull/3460), [#3462](https://github.com/grafana/k6/pull/3462), [#3463](https://github.com/grafana/k6/pull/3463), [#3478](https://github.com/grafana/k6/pull/3478), [#3479](https://github.com/grafana/k6/pull/3479), [#3480](https://github.com/grafana/k6/pull/3480), [#3481](https://github.com/grafana/k6/pull/3481), [#3482](https://github.com/grafana/k6/pull/3482), [#3483](https://github.com/grafana/k6/pull/3483), [#3484](https://github.com/grafana/k6/pull/3484), [#3485](https://github.com/grafana/k6/pull/3485), [#3495](https://github.com/grafana/k6/pull/3495).
+- [#3473](https://github.com/grafana/k6/pull/3473) Refinements to the release process.
+- Dependency updates across k6 [#3500](https://github.com/grafana/k6/pull/3500), [#3501](https://github.com/grafana/k6/pull/3501), [#3502](https://github.com/grafana/k6/pull/3502), [#3503](https://github.com/grafana/k6/pull/3503), [#3509](https://github.com/grafana/k6/pull/3509), [#3513](https://github.com/grafana/k6/pull/3513), [#3537](https://github.com/grafana/k6/pull/3537), [#3539](https://github.com/grafana/k6/pull/3539), [#3540](https://github.com/grafana/k6/pull/3540).
+- [#3489](https://github.com/grafana/k6/pull/3489) Assign pull-requests via CODEOWNERS instead of with a specific github action.
+- [#3496](https://github.com/grafana/k6/pull/3496) Check for security issues with a scheduled trivy scans.
+- [#3517](https://github.com/grafana/k6/pull/3517) Add some units tests to the cloadapi package. This is the first contribution by external contributor @nilskch. Thanks for this @nilskch :bow:.
+- [#3520](https://github.com/grafana/k6/pull/3520) Stop using deprecated by golang net.Dialer.DualStack option.
+- [#3526](https://github.com/grafana/k6/pull/3526) Some refactors to js package test around `open` and `require` and their handling of paths.
+- [#3527](https://github.com/grafana/k6/pull/3527) Generate test certificates for more tests during the test. This among other things fixes the tests on  MacOS.
+- [browser#1134](https://github.com/grafana/xk6-browser/pull/1134) adds a new error type when parsing objects.
+- [browser#1107](https://github.com/grafana/xk6-browser/pull/1107), [browser#1109](https://github.com/grafana/xk6-browser/pull/1109) refactor internals.
+
+
+## _Optional_ Roadmap
+
+_Discussion of future plans_
+
```

Comment:
I do intend on putting here that xk6-timers is going to go stable next release.
I don't think I will mention ESM as we have nothing new.
Anything else?

Reply:
All the big things we have in the queue have still high risks so I don't think we have more to mention with good confidence for now.

Reply:
Maybe let's add https://github.com/grafana/k6/pull/3092 ?

Reply:
@mstoykov I don't think it is so relevant for the users to be aware of it. Why would you mention it?

Reply:
From my side, nothing to add here. Do you want to add anything @inancgumus?

Reply:
Looks alright to me 👍 Thanks!

Reply:
@mstoykov can we drop the section or are you going to add something?

Reply:
See https://github.com/grafana/k6/pull/3541/files#r1466644809